Update teams-and-skypeforbusiness-coexistence-and-interoperability.md (#7851)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-and-skypeforbusiness-coexistence-and-interoperability.md

> Phone System isn't supported in Teams in **Islands** mode. **Islands** mode doesn't support Enterprise Voice client is Skype for Business.

Reviewer: ```suggestion
Phone System isn't supported in Teams in **Islands** mode. **Islands** mode doesn't support Enterprise Voice client in Skype for Business.
```

### Federation

Federation from Teams to another user using Skype for Business requires the Teams user to be homed online in Skype for Business. TeamsUpgradePolicy governs routing for incoming federated chats and calls. Federated routing behavior is the same as for same-tenant scenarios.

A federated chat can either be a native thread or an interop thread. See [Interop versus native conversation threads](#interop-versus-native-conversation-threads).

- If the receiver and sender are in TeamsOnly upgrade or Islands mode, the conversation will be a native chat experience which includes all the rich messaging and calling capabilities. To learn more, read [Native chat experience for external (federated) users in Teams](native-chat-for-external-users.md). 

> - If either of the conversation participants is on SfB* upgrade mode, the conversation remains an interop experience with text-only messages. The user interface exposes federated chats in a similar manner to same-tenant interop threads, except there is a note indicating the user is external.

Reviewer: ```suggestion
- If either of the conversation participants is in Skype for Business upgrade mode, the conversation remains an interop experience with text-only messages. The user interface exposes federated chats in a similar manner to same-tenant interop threads, except there is a note indicating the user is external.
```
